cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -7,6 +8,32 @@
 pytestmark = [test_requirements.report, pytest.mark.tier(3), pytest.mark.sauce]
 
 
+@pytest.fixture
+def create_custom_tag(appliance):
+    # cannot create a category with uppercase in the name
+    category_name = fauxfactory.gen_alphanumeric().lower()
+
```

Comment:
Question:
```cannot create a category with uppercase in the name```(Is it a bug or feature?)

Reply:
Neither I guess. But that is how the category name should be. If you use uppercase, it will throw a validation error.